tests/test_zipcode.py

```diff
@@ -0,0 +1,82 @@
+# tests.test_zipcode
+# Test the zipcode to latitutide,longitude lookup database.
+#
+# Author:   Benjamin Bengfort <benjamin@bengfort.com>
+# Created:  Mon May 08 12:35:21 2017 -0400
+#
+# Copyright (C) 2016 Bengfort.com
+# For license information, see LICENSE.txt
+#
+# ID: test_zipcode.py [] benjamin@bengfort.com $
+
+"""
+Test the zipcode to latitutide,longitude lookup database.
+"""
+
+##########################################################################
+## Imports and Fixtures
+##########################################################################
+
+import os
+import unittest
+
+from hanish.zipcode import ZipCodeDB
+from hanish.exceptions import HanishValueError
+
+
+FIXTURES = os.path.join(os.path.dirname(__file__), "..", "fixtures")
+ZIPCODES = os.path.join(FIXTURES, "ziplatlon.csv")
+
+
+##########################################################################
+## ZipCodeDB Tests
+##########################################################################
+
+class ZipCodeDBTests(unittest.TestCase):
+
+    def test_database_loading_lookup_and_close(self):
+        """
+        Integration test for loading and looking up zipcodes
+        """
+        zipdb = ZipCodeDB.load(ZIPCODES)
+
+        table = (
+            ('20001', (38.910353, -77.017739)),
```

Comment:
So the last zip with the leading zero could be an edge case, but you're right there is nothing materially different about the first 6 zipcodes. I still prefer to create tables e.g. [table driven tests](https://dave.cheney.net/2013/06/09/writing-table-driven-tests-in-go) because it allows me to quickly add boundary cases when they pop up during the operation of the software.